docs/content/preview/releases/techadvisories/ta-22935.md

```diff
@@ -0,0 +1,179 @@
+
+When server-side caching of sequences is enabled by setting the flag `ysql_sequence_cache_method = server`, it can cause errors for columns using the sequences if multiple databases are present in the cluster, or if a database is dropped and recreated. Columns using sequences as primary keys may incorrectly return duplicate key errors, and unique constraint violations may also occur if the column has a unique constraint.
+
+This issue arises because the same TServer cache entry is used across multiple databases, and sequence cache entries are never invalidated, remaining in memory till the TServer is restarted.
+
+## Mitigation
+
+If the cluster has `ysql_sequence_cache_method = server` enabled, then update the flag to use `ysql_sequence_cache_method = connection`.
+
+## Details
+
+By default, YugabyteDB supports caching of the sequence values in each YSQL connection. Capability to support the caching of the sequence value on yb-tserver was added to improve the performance of sequences. This feature can be turned on by setting the GFLAG ysql_sequence_cache_method = server.  The default value for this flag is connection.
+
+By default, YugabyteDB caches sequence values for each YSQL connection. To improve performance, YugabyteDB added support for caching sequence values on the yb-tserver. You can enable this feature by setting the flag `ysql_sequence_cache_method = server`. By default, this flag is set to "connection".
+
+For Tserver caching, the cache key is the OID of the sequence objects. PostgreSQL maintains unique OIDs within the database and thus multiple databases can allocate the same OID.  This can cause the sequences in two different databases to have the same OID. Since the cache key is OID, hence the different sequences in different databases will continue to use the same cache key which results in such sequences sharing the same cache in Tserver.
+
+For Tserver caching, the cache key is the OID (Object Identifier) of sequence objects. In PostgreSQL, each database maintains unique OIDs, but multiple databases can have the same OID. This means sequences in different databases might share the same OID. As the cache key is based on OID, different sequences in different databases will continue to use the same cache key which results in such sequences sharing the same cache in TServer.
+
+When `ysql_sequence_cache_method` is set to "server", sequence cache entries are never invalidated and remain in memory until the TServer shuts down. As a result, if a database containing sequences is dropped and then recreated, it might produce errors due to outdated entries from the dropped sequences.
+
+## Examples
+
+The following example demonstrates the same TServer cache entry being used across two different sequences in two different databases. To reproduce the following issue, set the cluster's flag `ysql_sequence_cache_method = server`.
+
+1. Create two databases seqtest1 and seqtest2 as follows:
+
+    ```sql
+    # create database seqtest1;
```

Comment:
format is wrong, the commands look like comments drop the #s
can drop the CREATE DATABASE output as well for these examples